### PR TITLE
Feature/sanitize

### PR DESCRIPTION
**Pull request by juherr**

### CODE REVIEW

**Comment by Evoluty** [comment]
Thanks for the changes. I totally agree with them.
Can you add a newline at the end of the .gitignore file ?
I think we need to fix the readme.md that I just added to be consistant with the new classes name "TimeZone" this could be included in the PR :) 
Nothing else to say this can be merged asap. Great one 👍

**Comment by Evoluty** [comment]
I think we need to add a new tag also for packagist :)

File: composer.json [hunk 1]
```diff
@@ -18,7 +18,10 @@
     "myclabs/php-enum": "^1.6"
   },
   "require-dev": {
-    "phpunit/phpunit": "^5"
+    "phpunit/phpunit": "^5",
+    "guzzlehttp/guzzle": "^6.3",
+    "php-http/guzzle6-adapter": "^2.0",
+    "http-interop/http-factory-guzzle": "^1.0"
```
Comment (Evoluty) [inline composer.json]: remove those dependencies please
to allow the CI to run well, add those dependencies in the travis.yml instead :)

Comment (Evoluty) [reply composer.json]: I'm checking if there is no way to avoid those

**Comment by juherr** [comment]
Done!

You're the owner, you merge! :)

**Pull request merged**